### DISCUSSION TITLE
cypress/vue: mount is not a function (component testing)

### DISCUSSION THREAD

**Discussion by relief-melone** [cypress-io/cypress#22526] [Questions and Help] [answered]
I am currently trying to integrate cypress in an existing vue2 application. I followed the guides to install cypress but I end up getting this error in my component tests

```
The following error originated from your test code, not from Cypress.

  > Object(...) is not a function

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Cypress could not associate this error to any specific test.

We dynamically generated a new test to display this failure.
```

Now I have been able to narrow it down to this line in the components.ts support file

```
Cypress.Commands.add('mount', mount);
```

I was able to add other custom functions without any problems. Also if I remove this line my tests don't crash. If I import mount directly in the test I end up with the same error again. Now I am not running into this problem when I set up a new project from scratch and I checked basically everything for differences but did not find any. I'd be happy to provide more info but am not sure where to look. Here's the current info about the versions:

node: 16.15.1
cypress: 10.2.0
vue: 2.6.11

https://stackoverflow.com/questions/68627950/error-in-cypress-object-is-not-a-function This user seems to have the same problem.  Unfortunately there are no responses yet. Would be great if someone had an idea what I am missing. Happy to provide more information

**Comment by ZachJW34** [comment, marked as answer]
@relief-melone Are you using the `cypress/vue2` import?

A simple test should look like:

```js
import MyComponent from './MyComponent.vue'
import { mount } from 'cypress/vue2'

it('should render', () => {
  mount(MyComponent)
})
```

**Reply by relief-melone** [reply]
Yes. Sorry in the end that one alone proved to be pretty basic. However it turned out starting out cypress for the first time with an existing project utilizing vue2/vuetify with composition api that relies heavily on provide inject was not too easy :D

If the complete solution is of any interest just let me know and I'll post it.

Thank you for the quick reply.
